src/main/java/com/sparrowwallet/sparrow/io/satochip/SatoCardTransport.java

```diff
@@ -0,0 +1,73 @@
+package com.sparrowwallet.sparrow.io.satochip;
+
+import com.sparrowwallet.drongo.Utils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.smartcardio.*;
+import javax.smartcardio.CardChannel;
+import java.util.List;
+import java.util.*;
+
+public class SatoCardTransport {
+    private static final Logger log = LoggerFactory.getLogger(SatoCardTransport.class);
+
+    private static final int SW_OKAY = 0x9000;
+
+    private final Card connection;
+
+    SatoCardTransport(byte[] applet_aid) throws CardException {
```

Comment:
for this constructor and methods below, is it wanted to have a package (default) visibility?